core/src/main/java/org/apache/calcite/schema/FunctionParameter.java

```diff
@@ -54,14 +52,19 @@
    * Returns whether this parameter is optional.
    *
    * <p>If true, the value of the parameter can be supplied using the DEFAULT
-   * SQL keyword, or it can be omitted from a function called using argument
-   * assignment, or the function can be called with fewer parameters (if all
-   * parameters after it are optional too).
+   * SQL keyword, or it can be omitted from a function called using argument assignment,
+   * or the function can be called with fewer
+   * parameters (if all parameters after it are optional too).
```

Comment:
Why do you change the format of comment? It is messed up now.

Reply:
undo, automatic code  formatter changed the comment.